src/v2/util/FormBuilder.js

```diff
@@ -118,6 +119,11 @@ const createInputOptions = function (appState) {
   return Form.extend(formObj);
 };
 
+const __getFormObj = function () {
+  return formObj;
+};
+
 module.exports = {
-  createInputOptions
+  createInputOptions,
+  __getFormObj
```

Comment:
we dont like to add code specifically for testing?

Reply:
yeah, I wanted some way to test if the formBuilder is setting up the inputs correctly. Would it help if we add an explicit comment here ?

Reply:
- Im thinking you could set `form.inputs` so that verify all the input options.
- or spy the `add` and `addInput` function, then render the form and verify the spy.